### ISSUE TITLE
build-storybook error

### ISSUE THREAD

**Issue by nicolaskruchten** [storybooks/addon-jsx#16]
I'm running into an issue with this addon when I run `build-storybook`, it's giving me

```
Building storybook ...
Failed to build the storybook
static/preview.89bc50f3418f0babaff2.bundle.js from UglifyJs
Invalid assignment [static/preview.89bc50f3418f0babaff2.bundle.js:44049,34]
```

Merely having this line `import JSXAddon from 'storybook-addon-jsx'` in `stories/index.js` causes this to happen... Any suggestions?

**Comment by nicolaskruchten** [comment]
I should add that 1) this is an awesome addon and 2) it works with `start-storybook` in my setup, just not with `build-storybook` :)

**Comment by nicolaskruchten** [comment]
How to replicate:

1. clone this: https://github.com/plotly/react-pivottable/tree/jsx-addon-bug
2. `npm install`
3. `npm run build-storybook`

**Comment by wcastand** [comment]
The response you got from the issue in storybook kind of answer to you i think no ? or you want something in particular ?

passing to babel-minify should fix the issue. The package `react-element-to-jsx-string` use a dependency that do not transpile to es5 anymore.

**Comment by nicolaskruchten** [comment]
Well, until the upstream patch is merged, this is still a big problem: we can't use this addon with `build-storybook` so let's keep this issue open until it's fixed upstream?

**Comment by ndelangen** [comment]
> passing to babel-minify should fix the issue. The package react-element-to-jsx-string use a dependency that do not transpile to es5 anymore.

This is on the way:
https://github.com/storybooks/storybook/pull/1733

**Comment by Hypnosphi** [comment]
> passing to babel-minify should fix the issue

But only if you're not targeting IE11

**Comment by markagrover** [comment]
I fixed the issue after lots of complication.

some plugins will add their own version of storybook as a dependency. Delete that if it pops up. Also this will go for most plugins I'm guessing. But in this plugin, I went into the plugin files and replaced a couple calls from @storybook/addons to @kadira/storybook-addons the changes were in files lib/index and lib/register. anywhere you see this pattern of calls to external files you will have to follow the errors and make the changes.

Happy coding!!!
